docs/sources/configure-client/grafana-agent/ebpf/configuration/_index.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "Configuration reference"
+menuTitle: "Configuration Reference"
+description: "Grafana Agent eBPF Profiling configuration reference"
+weight: 30
+---
+
+
+## Configuration
```

Comment:
The first heading on a page should be a first-level heading (# ) instead of a second-level (##). In this case, the first-level heading has to match the page title too. Configuration reference might not be clear enough as a page title for SEO values, too. 
```suggestion
title: "Configuration reference for eBPF"
menuTitle: "Configuration reference"
description: "Grafana Agent eBPF profiling configuration reference"
weight: 30
---

# Configuration reference for eBPF
```